src/components/Proposals/Proposals.js

```diff
@@ -54,28 +41,58 @@ const Proposals = React.memo(function Proposals() {
       >
         <LayoutLimiter>
           <Header primary="Proposals" />
-          {voteGroups.map(([groupName, votes]) =>
-            votes.length ? (
-              <ProposalCardGroup
-                title={groupName}
-                count={votes.length}
-                key={groupName}
-              >
-                {votes.map((vote) => (
-                  <ProposalCard
-                    key={vote.voteId}
-                    vote={vote}
-                    appAddress={disputableVotingApp.address}
-                    onProposalClick={handleProposalClick}
-                  />
-                ))}
-              </ProposalCardGroup>
-            ) : null
-          )}
+          <LoadingSection title="Loading proposals" loading={loading}>
+            <VotesLayout votes={votes} />
+          </LoadingSection>
         </LayoutLimiter>
       </LayoutGutter>
     </section>
   )
 })
 
+function VotesLayout({ votes }) {
+  const { disputableVotingApp } = useOrgApps()
+  const history = useHistory()
+
+  const handleProposalClick = useCallback(
+    (proposalId) => {
+      history.push(`/proposals/${proposalId}`)
+    },
+    [history]
+  )
+
+  const sortedVotes = useMemo(
+    () =>
+      [...votes].sort((a, b) => {
+        const dateDiff = b.endDate - a.endDate
+        // Order by descending voteId if there's no end date difference
+        return dateDiff !== 0 ? dateDiff : b.voteId - a.voteId
+      }),
+    [votes]
+  )
```

Comment:
Also sorting the votes in this PR, this was lifted straight from the client as I assume the same applies here.